dlrover/python/master/watcher/factory.py

```diff
@@ -45,3 +45,21 @@ def new_scale_plan_watcher(platform, job_name, namespace, job_uuid):
         return RayScalePlanWatcher(job_name, namespace, job_uuid)
     else:
         raise ValueError("Not support engine %s", platform)
+
+def new_elasticjob_watcher(args):
+    logger.info("New %s ElasticJob Watcher", args.platform)
```

Comment:
Better to use `f""` format for new code, although some historical implementations still use the `%` placeholder method.